core/src/main/resources/hudson/cli/Messages_it.properties

```diff
@@ -4,4 +4,4 @@ DeleteJobCommand.ShortDescription=Cancella un job
 OnlineNodeCommand.ShortDescription=Resume using a node for performing builds, to cancel out the earlier "offline-node" command.
 
 ClearQueueCommand.ShortDescription=Pulisce la coda di lavoro
-
+DisconnectNodeCommand.ShortDescription=Disconnects from a node
```

Comment:
It is taken from core/src/main/resources/hudson/model/Messages_it.properties, why we should remove it completely?